generator/generator.yml

```diff
@@ -561,3 +561,27 @@ modules:
         lookup: lgpPduRcpEntrySysAssignLabel
       - source_indexes: [lgpPduAuxSensorIndex]
         lookup: lgpPduAuxSensorSysAssignLabel
+
+# Wiener Power Supply Module MPod
+# 
+# http://www.wiener-d.com/sc/power-supplies/mpod--lvhv/mpod-crate.html
+# https://file.wiener-d.com/software/net-snmp/net-snmp-CompileForExtendedPrecision-2015-03-06.txt
+  wiener_mpod:
+    walk: # generator doesn't have 'get'
```

Comment:
This comment is misleading and should be removed - the right thing happens under the covers.